contracts/root/StakeManager.sol

```diff
@@ -79,31 +79,39 @@ contract StakeManager is StakeManagerInterface, RootChainable, Lockable {
   }
   
   function stake(uint256 amount, bytes data) public {
-    require(_validatorThreshold > nextValidatorList.currentSize(), "Next validator set full");
-    require(amount < MAX_UINT96, "Stay realistic!!");
-    require(stakers[msg.sender].epoch == 0, "No second time staking");
     stakeFor(msg.sender, amount, data);
   }
 
-  // function nextValidatorSetSize() public view  returns(uint256) {
-  //   return nextValidatorList.currentSize();
-  // }
+  function validatorSetSize() public view  returns(uint256) {
```

Comment:
No need to create this method. Each public variable automatically exposes method. Like `validatorSetSize` variable you will have `web3Contract.validatorSetSize()`